### PR TITLE
Minmaxrange null

### PR DESCRIPTION
**Pull request by gortiz**
Adds null support for minmaxrange operation. It assumes https://github.com/apache/pinot/pull/12215 is already merged

### CODE REVIEW

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java [hunk 1]
```diff
@@ -61,37 +61,29 @@ public GroupByResultHolder createGroupByResultHolder(int initialCapacity, int ma
   @Override
   public void aggregate(int length, AggregationResultHolder aggregationResultHolder,
       Map<ExpressionContext, BlockValSet> blockValSetMap) {
-    double min = Double.POSITIVE_INFINITY;
-    double max = Double.NEGATIVE_INFINITY;
 
     BlockValSet blockValSet = blockValSetMap.get(_expression);
+    MinMaxRangePair minMax = new MinMaxRangePair(Double.POSITIVE_INFINITY, Double.NEGATIVE_INFINITY);
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java:66]: Suggest adding an empty constructor to `MinMaxRangePair`

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java [hunk 2]
```diff
@@ -61,37 +61,29 @@ public GroupByResultHolder createGroupByResultHolder(int initialCapacity, int ma
...
-        double value = doubleValues[i];
-        if (value < min) {
-          min = value;
-        }
-        if (value > max) {
-          max = value;
+      forEachNotNull(length, blockValSet, (from, to) -> {
+        for (int i = from; i < to; i++) {
+          double value = doubleValues[i];
+          minMax.apply(value, value);
```
Comment (Jackie-Jiang) [inline pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java:73]: Suggest adding `void apply(double value)` to `MinMaxRangePair`

**Review by Jackie-Jiang — APPROVED** [review]

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java (hunk 1 above)
Comment (gortiz) [reply pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java]: Done

File: pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java (hunk 2 above)
Comment (gortiz) [reply pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/MinMaxRangeAggregationFunction.java]: Done

**Pull request merged**